bin/satpy_launcher.py

```diff
@@ -78,7 +78,10 @@ def process(msg, prod_list):
     job = message_to_job(msg, config)
     for wrk in config['workers']:
         cwrk = wrk.copy()
-        cwrk.pop('fun')(job, **cwrk)
+        try:
+            cwrk.pop('fun')(job, **cwrk)
+        except AbortProcessing:
+            LOG.info("Processing aborted for this scene")
```

Comment:
Can we add the error message to the info ?

Reply:
In think the plugin should handle that, it has more info on what happened. Although, I'll add the name of the plugin to the message.